scripts/cybershake/auto_submit.py

```diff
@@ -197,6 +199,28 @@ def submit_sl_script(script_name, **kwargs):
             {"submit_time": submitted_time},
             logger=task_logger,
         )
+
+    elif proc_type == const.ProcessType.IM_plot.value:
+        im_plot_template = (
+            "--export=CUR_ENV -o {output_file} -e {error_file} {script_location} "
+            "{csv_path} {rrup_or_station_path} {output_xyz_dir} {srf_path} {model_params_path} {mgmt_db_loc} {run_name}"
+        )
+        script = im_plot_template.format(
+            csv_path=os.path.join(sim_struct.get_IM_csv(sim_dir)),
+            rrup_or_station_path=glob.glob(os.path.join(sim_struct.get_fault_dir(root_folder, run_name.split('_')[0]), '*.ll'))[0],
+            output_xyz_dir=os.path.join(sim_dir, 'IM_plot'),
```

Comment:
All output plots should be in the verification folder rather than having their own folder.

IM_plot / plot_ts will have to move.